dev-infrastructure/svc-pipeline.yaml

```diff
@@ -263,9 +263,28 @@ resourceGroups:
       configRef: arobit.mdsd.image.registry
     - name: AROBIT_MDSD_ENABLED
       configRef: arobit.mdsd.enabled
-    dependsOn:
-    - svc
-    - global-output
```

Comment:
i think at least svc is required otherwise ev2 will try to install arobit before the cluster is ready

Reply:
ups. got removed in testing